stock_product_weight_on_receipt/models/stock.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class Product(models.Model):
```

Comment:
Change class name to `StockPackOperation`.

Reply:
Done